.github/workflows/ci.yml

```diff
@@ -0,0 +1,104 @@
+name: Continuous integration
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+          cache: pip
+
+      - name: Install dependencies
+        run: |
+          pip install -r requirements-dev.txt codecov
+          pip install -e .
+
+      - name: Run tests
+        run: pytest -ra -vv --doctest-modules --cov=.
+
+      - name: Run custom data source tests
+        run: python manage.py test --tag=custom_ds --settings=tests.test_app.settings_custom_ds
+
+      - name: Run custom pk tests
+        run: python manage.py test --tag=custom_pk_ds --settings=tests.test_app.settings_custom_pk_ds
+
+      - name: Coverage
+        run: codecov
+
+  build:
+    name: Build package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v1
```

Comment:
v4

Reply:
Well, I guess you can quickly check the versions used in other GH actions as well.

Reply:
I'll also make a task for updating munigeo